service/controller/drainer.go

```diff
@@ -82,10 +61,12 @@ func NewDrainer(config DrainerConfig) (*Drainer, error) {
 	var operatorkitController *controller.Controller
 	{
 		c := controller.Config{
-			Informer:     newInformer,
+			K8sClient:    config.K8sClient,
 			Logger:       config.Logger,
 			ResourceSets: resourceSets,
-			RESTClient:   config.K8sClient.K8sClient().CoreV1().RESTClient(),
+			NewRuntimeObjectFunc: func() runtime.Object {
+				return new(corev1.Pod)
```

Comment:
is it Pod? I thought it should be `KVMConfigs` in here.

Reply:
The drainer controller on KVM watches pods. You can check the resources being configured for the resource set.